lib/helpers.js

```diff
@@ -209,8 +209,29 @@ function buildStartCmd (startAppOptions, apiLevel) {
   return cmd;
 }
 
+async function getSdkToolsVersion () {
+  const androidHome = process.env.ANDROID_HOME;
+  if (!androidHome) {
+    log.errorAndThrow('ANDROID_HOME environment varibale is expected to be set');
+  }
+  const propertiesPath = path.resolve(androidHome, 'tools', 'source.properties');
+  if (!await fs.exists(propertiesPath)) {
+    log.warn(`Cannot find ${propertiesPath} file to read SDK version from`);
+    return;
+  }
+  const propertiesContent = await fs.readFile(propertiesPath, 'utf8');
+  const versionMatcher = new RegExp(/Pkg\.Revision=(\d+)\.?(\d+)?\.?(\d+)?/);
+  const match = versionMatcher.exec(propertiesContent);
+  // _.toInteger(undefined) returns zero, which is OK
+  if (match) return {
+    major: _.toInteger(match[1]),
```

Comment:
I get an error here : Encountered internal error running command: TypeError: _lodash2.default.toInteger is not a function. 

I use lodash 3.10.1